spec.yaml

```diff
@@ -170,6 +170,48 @@ paths:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
 
   /v4/users/{email}/:
+    get:
+      operationId: getUser
```

Comment:
`getUser` conflicts with the new operation defined in #82

Reply:
Ok, adjusted in #82, made the operationId there be `getCurrentUser`